app/src/main/java/com/android/partagix/ui/App.kt

```diff
@@ -78,14 +84,61 @@ class App(
           onItemCreated = { item -> inventoryViewModel.createItem(item) },
       )
   private val userViewModel = UserViewModel(db = db)
+  private val startOrEndLoanViewModel = StartOrEndLoanViewModel(db = db)
 
   @Composable
-  fun Create() {
+  fun Create(idItem: String? = null) {
 
     fusedLocationClient = LocationServices.getFusedLocationProviderClient(activity)
     ComposeNavigationSetup()
 
-    navigationActions.navigateTo(Route.BOOT)
+    if (idItem != null && FirebaseAuth.getInstance().currentUser != null) {
+      onQrScanned(idItem, FirebaseAuth.getInstance().currentUser!!.uid)
+    } else {
+      navigationActions.navigateTo(Route.BOOT)
+    }
+    /*
```

Comment:
open and end comment without anything in it

Reply:
Fixed